packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTConvert+RCTBPKCalendar.m

```diff
@@ -27,6 +30,38 @@ @implementation RCTConvert (RCTBPKCalendar)
                        @"range": @(BPKCalendarSelectionRange)
                    }),
                    BPKCalendarSelectionSingle, integerValue)
+
+RCT_ENUM_CONVERTER(RCTBPKColorBucketTextStyle, (@{
+                     @"default": @(RCTBPKColorBucketTextStyleDefault),
+                     @"light": @(RCTBPKColorBucketTextStyleLight),
+                     @"dark": @(RCTBPKColorBucketTextStyleDark)
+                 }),
+                 RCTBPKColorBucketTextStyleDefault, integerValue)
+
+RCT_ENUM_CONVERTER(RCTBPKDateMatcherType, (@{
+                   @"range": @(RCTBPKDateMatcherTypeRange),
+                   @"after": @(RCTBPKDateMatcherTypeAfter),
+                   @"before": @(RCTBPKDateMatcherTypeBefore),
+                   @"any": @(RCTBPKDateMatcherTypeAny)
+               }),
+               RCTBPKColorBucketTextStyleDefault, integerValue)
+
 RCT_ARRAY_CONVERTER(NSDate)
+RCT_ARRAY_CONVERTER(RCTBPKColorBucket)
+
++ (RCTBPKDateMatcher *)RCTBPKDateMatcher:(id)json {
+    RCTBPKDateMatcherType matcherType = [RCTConvert RCTBPKDateMatcherType:json[@"type"]];
+    NSArray<NSDate *> *dates = [RCTConvert NSDateArray:json[@"dates"]];
```

Comment:
I don't think this is still handling the timezone issues correctly

Reply:
That's handled in `RCTBPKCalendar setRct_colorBuckets`